swift/swift.py

```diff
@@ -16,12 +16,95 @@
 import importlib
 import importlib.util
 from contextlib import contextmanager
+from dataclasses import dataclass, asdict
+from typing import Any, Iterable, Mapping, Self
 
 from PyQt5.QtCore import QObject, pyqtSlot, Qt
 from PyQt5.QtWidgets import QApplication, QMainWindow, QDockWidget
 
 from swift.bus import Bus
 
+@dataclass
+class AppInfo:
+    """Information required to create an app.
+    
+    Fields:

```

Comment:
It was hard for me to choose the word 😄
I decided to use "Fields" because `dataclasses` use the word [`field`](https://docs.python.org/3/library/dataclasses.html#dataclasses.field) for the data class attributes.

In addition, they are not "Args" but "Attributes".

Reply:
This decision is valid exclusively for data classes.
If you have other opinions, freely suggest other options.

Reply:
If you think `Fields` is more appropriate, I'm ok!